### DISCUSSION TITLE
using SPARQL Anything to transform relational DBs into RDF

### DISCUSSION THREAD

**Discussion by justin2004** [SPARQL-Anything/sparql.anything#367] [General]
I just wrote a blog post on it!

https://github.com/justin2004/weblog/blob/master/relational_as_graph/README.md

**Comment by luigi-asprino** [comment]
Thanks Justin, nice indeed. Let me link it from the [tutorials page](https://github.com/SPARQL-Anything/sparql.anything/blob/v0.9-DEV/TUTORIALS.md)

**Comment by enridaga** [comment]
This is soo cool Justin, I love it!

**Comment by StuartJAtkinson** [comment]
This is amazing

**Comment by VladimirAlexiev** [comment]
hi @justin2004 a couple of suggestions for improvement:
- use `fx:properties fx:use-rdfs-member true` in the service clause so it uses `rdfs:member` as connecting prop rather than ugly props ilke `<http://www.w3.org/1999/02/22-rdf-syntax-ns#_6>`
(SPARQL-Anything has the magic predicate `fx:anySlot` to consume any `rdf:_N` but many RDF frameworks/repos don't)
- use `fx:entity(ex:, "Warehouse_", ?warehouse)` rather than `IRI(concat(str(ex:), "Warehouse_", ?warehouse))` for brevity
